examples/node/register-contract.js

```diff
@@ -0,0 +1,32 @@
+const DashJS = require('dash');
+
+const sdkOpts = {
+  network: 'testnet',
+  mnemonic:'your mnemonic here'
+};
+const sdk = new DashJS.Client(sdkOpts);
+
+const registerContract = async function () {
+  await sdk.isReady();
+  let platform = sdk.platform;
```

Comment:
const { platform } = client;

Reply:
Fixed in this PR : https://github.com/dashevo/DashJS/pull/39